scanner/updater/export.go

```diff
@@ -66,21 +66,28 @@ func Export(ctx context.Context, outputDir string) error {
 	if err != nil {
 		return err
 	}
-	outOfTree := append([]driver.Updater{}, manualSet.Updaters()...)
-	opts = append(opts, []updates.ManagerOption{updates.WithOutOfTree(outOfTree)})
+	opts = append(opts, []updates.ManagerOption{
+		// This is required to prevent default updaters from running.
+		updates.WithEnabled([]string{}),
```

Comment:
Were we downloading duplicate vulnerabilities ? 
`updates.WithEnabled([]string{})` means no default updater is enabled?

Reply:
Yes, the import at the top `_ "github.com/quay/claircore/updater/defaults"` registers all the default updaters. For `rhel-custom`, `manual`, and `nvd`, we never set `WithEnabled`, so ClairCore used all the default updaters, which blew up the vuln data quite a bit. This PR sets `WithEnabled([]string{})`, as an empty slice is seen as not wanting any default updaters. See https://github.com/quay/claircore/blob/2426565cf3edbe53ee07fe98d4e6305618134ec6/matchers/options.go#L13

Note: I kept the import in there, as it allows us to just write the updater's name instead of having to include the factory information

Reply:
Thank you! The PR looks good to me.